### ISSUE TITLE
Uncaught TypeError: t.termSetFont is not a function

### ISSUE THREAD

**Issue by DonielMoins** [nitrous-io/nitrogen-dist#15]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.5.1
**System**: Microsoft Windows 10 Home Single Language
**Thrown From**: [nitrogen](https://github.com/nitrous-io/nitrogen-dist) package, v0.1.4
### Stack Trace

Uncaught TypeError: t.termSetFont is not a function

```
At C:\Users\theshark\.atom\packages\nitrogen\lib\components\nitrogen-shell.js:1

TypeError: t.termSetFont is not a function
    at t.setFont (C:\Users\theshark\.atom\packages\nitrogen\lib\components\nitrogen-shell.js:1:2859)
    at t._createClass.value (C:\Users\theshark\.atom\packages\nitrogen\lib\components\nitrogen-shell.js:1:5121)
    at EventEmitter.n (C:\Users\theshark\.atom\packages\nitrogen\lib\components\nitrogen-shell.js:1:4654)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:169:7)
```
### Commands

```
 15x -1:01.5.0 core:backspace (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db",
      ".unison",
      "CVS",
      ".AppleDouble",
      ".LSOverride",
      ".Spotlight-V100",
      ".Trashes",
      "desktop.ini",
      "*.lock",
      "*.sock",
      "*.pid",
      "*.sw?",
      ".unison*",
      "*~"
    ],
    "themes": [
      "atom-dark-ui",
      "one-dark-syntax"
    ]
  },
  "nitrogen": {
    "showWelcomeOnStartup": false
  }
}
```
### Installed Packages

``` coffee
# User
nitrogen, vundefined

# Dev
No dev packages
```

then this
C:\Users\user.atom\packages\nitrogen\lib\components\nitrogen-shell.js:1
Hide Stack Trace
TypeError: t.termSetFont is not a function
    at t.setFont (C:\Users\user.atom\packages\nitrogen\lib\components\nitrogen-shell.js:1:2859)
    at t._createClass.value (C:\Users\user.atom\packages\nitrogen\lib\components\nitrogen-shell.js:1:5121)
    at EventEmitter.n (C:\Users\user.atom\packages\nitrogen\lib\components\nitrogen-shell.js:1:4654)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:169:7)

**Comment by styner32** [comment]
This issue should be fixed in v0.1.5. Does anyone still have this issue?

**Comment by nitrogen-ide** [comment]
-- Please reply above this line --

Hello,

We received your Nitrous support request! One of our support engineers
will respond to you as soon as possible. In the meantime, some of
these documents might be helpful: 

https://community.nitrous.io/docs [1]
https://community.nitrous.io/docs/problems-previewing-my-app [2]
https://community.nitrous.io/docs/ide-overview [3]
https://community.nitrous.io/docs/billing-pricing-faq [4]

Thanks!

Nitrous Support

## Links:

[1] https://community.nitrous.io/docs
[2] https://community.nitrous.io/docs/problems-previewing-my-app
[3] https://community.nitrous.io/docs/ide-overview
[4] https://community.nitrous.io/docs/billing-pricing-faq

**Comment by DonielMoins** [comment]
> This issue should be fixed in v0.1.5. Does anyone still have this issue?

Nope, I'm closing this now. Surprised it was left open for... what like... 4 years now?
oh god no its been 6 years... Yikes
